internal/concierge/impersonator/impersonator_test.go

```diff
@@ -1932,13 +1927,11 @@ func newRequest(t *testing.T, h http.Header, userInfo user.Info, event *auditint
 		ctx = request.WithUser(ctx, userInfo)
 	}
 
-	ae := &auditinternal.Event{Level: auditinternal.LevelMetadata}
+	ctx = audit.WithAuditContext(ctx)
 	if event != nil {
-		ae = event
+		ac := audit.AuditContextFrom(ctx)
```

Comment:
So, if `event` is `nil`, nothing gets created, but this still works?  Though we said that `Events` are no longer pointers?

Reply:
But `ac.Event = *event` is a pointer, and seems to work?

Reply:
The type for `ac` has changed in v0.28.0:

* https://github.com/kubernetes/apiserver/blob/v0.27.0/pkg/audit/context.go#L43
* https://github.com/kubernetes/apiserver/blob/v0.28.0/pkg/audit/context.go#L43